client/src/app/api/rest.ts

```diff
@@ -358,6 +358,24 @@ export function getTaskByIdAndFormat(
     });
 }
 
+export function getTasksByIds(
+  ids: number[],
+  format: "json" | "yaml"
+): Promise<Task[]> {
+  const isYaml = format === "yaml";
+  const headers = isYaml ? { ...yamlHeaders } : { ...jsonHeaders };
+  const responseType = isYaml ? "text" : "json";
+
+  return axios
+    .post<Task[]>(`${TASKS}/multiple`, ids, {
+      headers: headers,
+      responseType: responseType,
+    })
```

Comment:
Is there a reasons this does not use the existing _List_ endpoint with a filter?
Example:
```
/tasks?filter=id:(1,2,3)
```